RickAndMortyFunApp/TabBarController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TabBarController.swift
+//  RickAndMortyFunApp
+//
+//  Created by Rusłan Chamski on 24/02/2019.
+//  Copyright © 2019 Rusłan Chamski. All rights reserved.
+//
+
+import UIKit
+
+class TabBarController: UITabBarController {
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupViewControllers()
+    }
+
+    fileprivate func setupViewControllers() {
+
+        let charcterViewController = CharacterViewController()
+        let characterNavController = UINavigationController(rootViewController: charcterViewController)
+
+        let locationViewController = LocationViewController()
+        let locationNavController = UINavigationController(rootViewController: locationViewController)
+
+        let episodeViewController = EpisodeViewController()
+        let episodeNavController = UINavigationController(rootViewController: episodeViewController)
+
+        let searchViewController = SearchViewController()
+        let searchNavController = UINavigationController(rootViewController: searchViewController)
+
+        self.viewControllers = [characterNavController, locationNavController, episodeNavController, searchNavController]
+
+
+        let myTabBarItem1 = (self.tabBar.items?[0])! as UITabBarItem
+        myTabBarItem1.title = "Characters"
```

Comment:
This string ideally should be localised. Great opportunity to try it. Actually it's very simple. For now, let's just use NSLocalizedString instead